bin/hdfcoinc/pycbc_multiifo_sngls_findtrigs

```diff
@@ -1,20 +1,24 @@
 #!/usr/bin/env python
```

Comment:
A general comment: This is definitely going to conflict with #3345. My take is the easiest approach is to merge that first, and then rebase this on top of that.